Add files via upload

Virginrat opened the pull request.

Commit:
Sprint_4

Virginrat committed:
Add files via upload

pomegranate282922 commented on src/main/java/ru/qascooter/praktikumservices/pageobjects/ScooterOrder.java:
```diff
@@ -0,0 +1,141 @@
+import static com.codeborne.selenide.Selectors.byText;
+
+public class ScooterOrder {
+    public static final String URL = "https://qa-scooter.praktikum-services.ru/";
```
Отлично что вынесены урлы

pomegranate282922 commented on src/test/java/tests/AccordeonTests.java:
```diff
@@ -0,0 +1,29 @@
+
+        //метод для всех вопросов и получения текстов всех ответов
+        mainPageQuestions.clickOnQuestionsAndHaveAnswersTexts();
+    }
```
Каждая проверка должна быть независимой. Нужно применить параметризацию 
Нужно исправить

pomegranate282922 commented on src/test/java/tests/AccordeonTests.java:
```diff
@@ -0,0 +1,29 @@
+}
+
+    @After
+    public void tearDown() {
```
Общие начала и концы всех тестовых классов  лучше вынести в общий класс предок 
Можно улучшить

pomegranate282922 commented on src/test/java/tests/ScooterOrderTests.java:
```diff
@@ -0,0 +1,67 @@
+       //подтверждение
+       mainPageOrderByPageMiddleButton.clickConfirmOrderButton(); //клик по кнопке подтверждения заказа
+       mainPageOrderByPageMiddleButton.orderTitleIsDisplayed(); //поиск заголовка "Заказ оформлен"
+    }
```
Все хорошо, но нужно применить параметризацию 
Нужно исправить

pomegranate282922 reviewed (commented):
Работа хорошая! Все тесты реализованы. 
замечания: в обоих тестовых классах нужно применить параметризацию

Commit:
Merge branch 'develop' of https://github.com/Virginrat/Sprint_4 into develop

Commit:
Sprint_4

обновление согласно полученным комментариям + исправлены имена переменных для полей формы
